docs/v2/2_installation.md

```diff
@@ -9,31 +9,27 @@ sidebar_label: Installation
 
 :::info
 
-In this section, we will go through the steps necessary to install the web3.unity SDK into the Unity3D game engine.
+In this section, we will go through the steps necessary for installing the web3.unity SDK into the Unity3D game engine. We will follow the installation with an example of how to implement a sample web3 login screen.
```

Comment:
> Unity3D game engine
^ I know this was here before, but maybe we change this to Unity Game Engine since it supports more than just 3D.

(change in all places unless explicitly stated for 3D rendering which I don't think we have) 

@nftpixels to confirm

Reply:
I see, it seems that Unity3D is being used in this document as an example of building specifically a 3D game. So maybe it makes sense to have this after all.